### PR TITLE
Update essentials markdown documents to MarkBind

### PR DESCRIPTION
**Pull request by lhw-1**
Updated markdown documents under `src/essentials` to MarkBind-supported format.

### CODE REVIEW

File: docs/essentials/component-basics.md [hunk 1]
```diff
@@ -192,7 +192,7 @@ export default {
 </script>
 
 <template>
-  <h4>{{ title }}</h4>
+  <h4>{{ '{{' }} title }}</h4>
```
Comment (itsyme) [inline docs/essentials/component-basics.md]: I think this should be `<h4>{{ '{{' }} title {{ '}}' }}</h4>` if I am not wrong

File: docs/essentials/reactivity-fundamentals.md [hunk 2]
```diff
@@ -472,20 +472,20 @@ const { foo } = object
...
 
 ```vue-html
-{{ foo + 1 }}
+{{ '{{' }} foo + 1 {{ '}}' }}
 ```
 
 Now the render result will be `2`.
 
-One thing to note is that a ref will also be unwrapped if it is the final evaluated value of a text interpolation (i.e. a <code v-pre>{{ }}</code> tag), so the following will render `1`:
+One thing to note is that a ref will also be unwrapped if it is the final evaluated value of a text interpolation (i.e. a <code v-pre>{{ '{{' }} }}</code> tag), so the following will render `1`:
```
Comment (itsyme) [inline docs/essentials/reactivity-fundamentals.md]: I think this part should be as follows!
`{{ '{{ }}' }}`

File: docs/essentials/component-basics.md (hunk 1 above)
Comment (lhw-1) [reply docs/essentials/component-basics.md]: Thanks for finding this! Will edit now :+1:

File: docs/essentials/reactivity-fundamentals.md (hunk 2 above)
Comment (lhw-1) [reply docs/essentials/reactivity-fundamentals.md]: Thanks for finding this as well!

**Pull request merged**